lib/packExternalModules.js

```diff
@@ -153,28 +153,28 @@ function findExternalOrigin(issuer) {
 }
 
 function getExternalModules(stats) {
-  if (!stats.compilation.chunks) {
-    return [];
-  }
-
-  const externals = new Set();
-  for (const chunk of stats.compilation.chunks) {
-    if (!chunk.modulesIterable) {
-      continue;
-    }
+  if (stats.compilation.chunks) {
```

Comment:
I personally would have kept the early return instead of putting the main body of code inside an `if` block